src/sql/parts/notebook/cellViews/textCell.component.ts

```diff
@@ -72,9 +74,9 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 	 * Sanitizes the data to be shown in markdown cell
 	 */
 	private updatePreview() {
-		if (this._content !== this.cellModel.source) {
+		if (this._content !== this.cellModel.source || this.cellModel.source.length === 0) {
 			if (!this.cellModel.source && !this.isEditMode) {
-				(<HTMLElement>this.output.nativeElement).innerHTML = localize('doubleClickEdit', 'Double-click to edit');
+				(<HTMLElement>this.output.nativeElement).innerHTML = localize('doubleClickEdit', 'Select the cell to edit');
```

Comment:
If it's 'select cell' instead of 'double click', would be better to change the key name as well from doubleClickEdit to selectCellEdit